etc/profile

```diff
@@ -61,6 +61,13 @@ MINGW64)
   ACLOCAL_PATH="${MINGW_MOUNT_POINT}/share/aclocal:/usr/share/aclocal"
   MANPATH="${MINGW_MOUNT_POINT}/local/man:${MINGW_MOUNT_POINT}/share/man:${MANPATH}"
   ;;
+ARM64)
+  MINGW_MOUNT_POINT="${MINGW_PREFIX}"
+  PATH="${MINGW_MOUNT_POINT}/bin:/mingw32/bin:${MSYS2_PATH}${ORIGINAL_PATH:+:${ORIGINAL_PATH}}"
```

Comment:
We don't need `mingw32/bin` in here anymore if we choose to go down the path from https://github.com/git-for-windows/MINGW-packages/pull/44 and rename `mingw32` to `arm64` prior to copying the arm64 executables into it:

![image](https://user-images.githubusercontent.com/17739158/103213261-6c946880-490d-11eb-9421-836fe8b9db15.png)

Curious to hear your thoughts on this

Reply:
That's @dscho 's idea.

Reply:
I would strongly recommend _not_ renaming `mingw32` to `arm64`, but instead _adding_ the latter. Then, `PATH` can be configured in such a way that the binaries from `arm64/bin/` are preferred over those from `mingw32/bin/`, and we're free to only provide a subset of the `.exe` files for ARM64 (with the rest just being used in their i686 form).

Reply:
Makes sense. Just applied that in https://github.com/git-for-windows/MINGW-packages/pull/44; `arm64` comes first in the path, then `mingw32`.